packages/dashboard/webpack.config.ts

```diff
@@ -0,0 +1,103 @@
+import * as webpack from "webpack";
+import type WebpackDevServer from "webpack-dev-server";
+import HtmlWebpackPlugin from "html-webpack-plugin";
+import EslintWebpackPlugin from "eslint-webpack-plugin";
+import TerserWebpackPlugin from "terser-webpack-plugin";
+import path from "path";
+
+const isProduction = process.env.NODE_ENV === "production";
+
+const progressHandler = (function () {
+  let lastLoggedPercent = -1;
+  const minInterval = 15;
+
+  return {
+    log(percent: number, _message: string, ..._args: string[]) {
+      percent = Math.round(percent * 1000) / 10;
+      if (lastLoggedPercent < 0 || percent - lastLoggedPercent >= minInterval) {
+        lastLoggedPercent = percent;
+        console.log(`Compiling: ${percent}%`);
+      } else if (percent === 100) {
+        console.log(`Finished compiling`);
+      }
+    }
+  };
+})();
+
+const config: webpack.Configuration = {
+  mode: isProduction ? "production" : "development",
+  entry: "./src/index.tsx",
+  output: {
+    filename: "bundle.js",
+    path: path.resolve("build")
+  },
+  optimization: {
+    minimize: isProduction,
+    minimizer: [
+      new TerserWebpackPlugin({
+        terserOptions: {
+          compress: {
+            drop_console: isProduction
+          }
+        }
+      })
+    ]
+  },
+  resolve: {
+    extensions: [".ts", ".tsx", ".js", ".jsx"],
+    alias: {
+      src: path.resolve("src")
+    },
+    fallback: {
+      crypto: require.resolve("crypto-browserify"),
+      os: require.resolve("os-browserify/browser"),
+      path: require.resolve("path-browserify"),
+      stream: require.resolve("stream-browserify"),
+      vm: require.resolve("vm-browserify")
+    }
```

Comment:
Do we actually _need_ any of these packages in the dashboard frontend? If not, maybe we should use `false` instead of `require.resolve(polyfillPackageName)`. That would help reduce the weight of the dependency tree.

Reply:
I believe so. The graph goes `dashboard -> decoder -> web3-utils`. The specifics of how `web3-utils` uses node core modules I'm not sure, but I imagine setting these to `false` would render it unpredictable / unworkable.